tools/profiling/templates/functions/php-version.tpl

```diff
@@ -0,0 +1,9 @@
+{function name=php_version}
+  {if version_compare($version, '5.6') < 0}
+    <span class="danger">{$data} (Upgrade strongly recommended)</span>
+  {elseif version_compare($version, '7.1') < 0}
+    <span class="warning">{$data} (Consider upgrading)</span>
+  {else}
+    <span class="success">{$data} (OK)</span>
+  {/if}
```

Comment:
May be add {elseif version_compare($version, '7.4') > 0}

Reply:
It's not needed, you can't install PrestaShop with a version > 7.4 :sweat_smile: 
I just backport what was in the original template, but I maybe can remove the 5.6 check :thinking:

Reply:
Ok. Good answer.